packages/lit-dev-content/site/blog/2022-02-07-eleventy.md

```diff
@@ -0,0 +1,181 @@
+---
+tags: blogPosts
+layout: blog-post.html
+title: "Eleventy + Lit"
+summary: "Announcing a new Lit Labs Eleventy plugin for static pre-rendering of Lit components"
+date: 2022-02-07
+author:
+  - al-marks
+---
+
+The Lit team is pleased to announce the first release of
```

Comment:
maybe "first Lit Labs release" or "first experimental/preview release" to drive home the point in the lede before the alert.